src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -458,7 +462,8 @@ def add_agent_manager(self, config_file=None, common_cfg=None, timeout=30):
         self.agent_managers.append(
             DaosAgentManager(agent_cmd, self.manager_class))
 
-    def add_server_manager(self, config_file=None, common_cfg=None, timeout=90):
+    def add_server_manager(self, config_file=None, dmg_config_file=None,
```

Comment:
It's better to add description for dmg_config_file.

Reply:
Done.